portals/publisher/package.json

```diff
@@ -31,15 +31,18 @@
         "@apidevtools/swagger-parser": "^10.0.1",
         "@asyncapi/parser": "1.3.2",
         "@asyncapi/react-component": "^0.19.0",
+        "@autocomplete/material-ui": "0.0.17",
         "@hapi/hapi": "^18.4.1",
         "@material-ui/core": "^4.11.3",
         "@material-ui/icons": "^4.9.1",
+        "@material-ui/lab": "^4.0.0-alpha.60",
         "@types/react": "^17.0.37",
         "async-mutex": "^0.1.3",
         "async-react-component": "^0.7.0",
         "autosuggest-highlight": "^3.1.1",
         "babel-plugin-formatjs": "^9.0.4",
         "base64url": "3.0.1",
+        "bootstrap": "^5.1.3",
```

Comment:
Why do we need the bootsrap library ?

Reply:
Removed from https://github.com/wso2/apim-apps/pull/144/commits/41d370ff08de94c0c8bdfe0e408a0c56d137b94b